### PR TITLE
Update Operator-SDK meetings information

### PR DESCRIPTION
**Pull request by asmacdo**

### CODE REVIEW

**Review by kensipe — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -17,22 +17,28 @@ Discuss on:
 
 ## Meetings
 
-### Operator Framework Special Interest Group
-
-* Monthly, on 3rd Tuesday 16:00 UTC [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:00&tz=UTC)
-  * [Meeting link](https://bluejeans.com/8168972123)
-  * [Agenda](https://github.com/operator-framework/community/projects/2)
-
 ### Operator SDK
```
Comment (estroz) [inline README.md:20]:
```suggestion
### Operator SDK monthly meeting
```

File: README.md [hunk 2]
```diff
@@ -17,22 +17,28 @@ Discuss on:
...
 
 * Wednesdays at 09:00 PT (Pacific Time) (monthly - first Wednesday of the month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting Link](https://zoom.us/j/8415370125)
+  * Meeting Password: 77777
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1ujWb-rSJ4JWeHLVxK0WS5ZuSJgeESG42MDeYjSl9Q6U/edit#)
   * Meeting recordings (coming soon!)
   
-### Operator SDK community issue triage/grooming
+### Operator SDK community issue triage
```
Comment (estroz) [inline README.md:28]: "Community" is implied I think
```suggestion
### Operator SDK issue triage
```

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**